src/languages/sqf.js

```diff
@@ -2059,12 +2179,12 @@ export default function(hljs) {
     'setTaskResult',
     'setTaskState',
     'setTerrainGrid',
+    'setTerrainHeight',
     'setText',
-    'setThrottleRTD',
     'setTimeMultiplier',
+    'setTiParameter',
```

Comment:
```suggestion
    'setTIParameter',
```
https://community.bistudio.com/wiki/setTIParameter